providers/flagd/src/main/java/dev/openfeature/contrib/providers/flagd/resolver/process/model/FlagParser.java

```diff
@@ -110,8 +111,9 @@ private static String transposeEvaluators(final String configuration) throws IOE
 
             while (evalFields.hasNext()) {
                 final String evalName = evalFields.next();
-                // first replace brackets
-                final String replacer = REG_BRACKETS.matcher(evaluators.get(evalName).toString()).replaceAll("");
+                // first replace outmost brackets
+                final String evaluator = evaluators.get(evalName).toString();
+                final String replacer = evaluator.substring(1, evaluator.length() - 1);
```

Comment:
can we assume the first char is always a bracket?

Reply:
Yes this is a safe assumption. There won't be spaces from deserialization.